analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/InitializationVectorShouldBeRandom.CSharp9.cs

```diff
@@ -43,7 +43,7 @@ public int Property
         init
         {
             AesCng aes = new AesCng();
-            aes.CreateEncryptor(); // FIXME Non-compliant
+            aes.CreateEncryptor(); // FN
```

Comment:
Why was this marked as Noncompliant before?

Reply:
It was like that in the old implementation - but it's wrong indeed.
I changed the example to be non-compliant so we are sure that issue is raised in init.
I will unmark it in the old implementation too.